packages/manager/modules/pci/src/components/project/regions-list/index.js

```diff
@@ -14,6 +15,7 @@ angular
     'pascalprecht.translate',
   ])
   .component('pciProjectRegionsList', component)
+  .service('PciProjectRegions', service)
```

Comment:
Is that make sense to you?
```suggestion
  .service('pciProjectRegions', service)
```

Reply:
How do we differentiate a `service` from a `component` in that case ? Because none of them is suffixed by their type according to our logic so far, the difference was made with that caps letter

Reply:
LGTM ;)

Reply:
@marie-j John Papa uses the `Service` suffix as `UpperCamelCase` is used by Javascript by data types and function constructors (`RegExp`, `TypeError`, `XMLHttpRequest` for example)

i.e.: Douglas Crockford's coding convention (https://www.crockford.com/code.html)

Reply:
As spoken with @antleblanc earlier, it makes sense to keep it with `CamelCase`, as it's a singleton there's only one possible instance

Reply:
`TileCase` !== `camelCase` to be more precise.

Here the discussion has been marked as resolved because it could be ambiguous due to the fact the both component and service has the same name.